### PR TITLE
Added functionality for replay game, when finished

### PR DESCRIPTION
**Pull request by dianamarin93**

### CODE REVIEW

**Review by GabrielCorlan — COMMENTED** [review]
It would be better to delimit main() method of other methods trough spaces

File: Paper_Rock_Scissors_Game/Program.cs [hunk 1]
```diff
@@ -118,22 +121,35 @@ public static void Play_Game()
...
-                Console.WriteLine($"The winner of the game is the computer, which has the score {computerScore}");
-            }
-            else if (playerScore > computerScore)
-            {
-                Console.WriteLine($"The winner of the game is the player, which has the score {playerScore}");
-            }
-            else
-            {
-                Console.WriteLine("Equality!");
-            }
```
Comment (GabrielCorlan) [inline Paper_Rock_Scissors_Game/Program.cs]: too many spaces

Comment (dianamarin93) [reply Paper_Rock_Scissors_Game/Program.cs]: Done

**Comment by GabrielCorlan** [comment]
LGTM

**Review by GabrielCorlan — APPROVED** [review]

**Pull request merged**